x-pack/plugins/fleet/server/services/epm/kibana/assets/install.ts

```diff
@@ -329,14 +335,16 @@ export async function installKibanaSavedObjects({
   if (toBeSavedObjects.length === 0) {
     return [];
   } else {
+    const readStream = createListStream(toBeSavedObjects);
```

Comment:
I was hopeful that moving this out of the retry block would help with some memory usage here but it does seem to have made much of a difference

![image](https://github.com/elastic/kibana/assets/6766512/eb76a473-4a45-4700-ac31-f4d7feaff37c)